packages/showcases/core/csf/components/indicators/rating.csf.js

```diff
@@ -71,4 +71,10 @@ export const argTypes = {
     defaultValue: false,
     control: { type: 'boolean' },
   },
+  ariaLabel: {
+    type: { name: 'string', required: false },
+    description: 'Add aria label for accessibility',
+    defaultValue: '{0} star out of 5',
```

Comment:
defaultValue: {}

Reply:
To discuss with the comment above

Reply:
I wonder if, in terms of DX, it would not be better to have a props API more "agnostic". Things could be more transparent and the (dev) user should not necessarily have to know the aria standard (I agree it is better if they do). Here i would just add a more common "label" prop then, that we would also use internally in the aria attributes.

I agree that it should be required in all cases, without default value then. The component should not make assumptions about its context of use.